documents/description.md

```diff
@@ -1,9 +1,40 @@
 # Project description
 
 ## Motivation
+Scala Collections provide a wide range of useful functions for operating on 
+lists of objects. These functions include _map_, _filter_, _fold_ an many more. 
+SQL query languages however, despite being representable by a collection of 
+tuples, lack most of these very useful functions. This can result in many over-
+complicated SQL queries. The goal of this project is to augment the experience 
+of working with and manipulating SQL data by providing the above mentioned 
+Collection functions for SQL databases.
```

Comment:
Quite a bit more detail here could be helpful, especially for the uninitiated. For instance, it could be useful to include a specific example of a query that could be improved with functional features.